### ISSUE TITLE
Cannot authenticate/reauthenticate/logout with v 13.13.2

### ISSUE THREAD

**Issue by ken-salterdiazsolutions** [firebase/firebase-tools#7447]
### [REQUIRED] Environment info

firebase 13.13.2

**firebase-tools:**

macos

**Platform:**

### [REQUIRED] Test case

firebase login --reauth

Error: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci

Having trouble? Try firebase [command] --help
firebase logout

Error: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci

Having trouble? Try firebase [command] --help

### [REQUIRED] Steps to reproduce

try to issue commands firebase login, firebase login --reauth, firebase logout

### [REQUIRED] Expected behavior

The commands work as expected

### [REQUIRED] Actual behavior

The commands fail with the Authentication Error message

WORKAROUND:  Uninstalled 13.13.2, installed 13.13.1, all commands work as expected.

**Comment by liambolling** [comment]
+1 - I can't login to firebase (or logout) in the CLI at all. Forced to downgrade.

**Comment by glr-shaun** [comment]
I also cannot login and logout from yesterday.
```
docker compose run --rm firebase firebase login --no-localhost
WARN[0000] network server: network.external.name is deprecated in favor of network.name

Error: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci

**Comment by serpro69** [comment]
Ah, been wracking my brain with why it suddenly stopped working...

```
➜ firebase --debug login --no-localhost
[2024-07-15T08:27:21.576Z] > refreshing access token with scopes: []
[2024-07-15T08:27:21.578Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[2024-07-15T08:27:21.578Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[2024-07-15T08:27:21.733Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 400
[2024-07-15T08:27:21.733Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]

Error: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci

➜ firebase --debug login --reauth --no-localhost
[2024-07-15T08:27:37.140Z] > refreshing access token with scopes: []
[2024-07-15T08:27:37.142Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[2024-07-15T08:27:37.142Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[2024-07-15T08:27:37.271Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 400
[2024-07-15T08:27:37.271Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]

Error: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci

Having trouble? Try firebase [command] --help


```

**Comment by aalej** [comment]
Hey folks, apologies for the issue this has caused. We received a similar report in #7442, and our team has a fix in #7445 that should be available in the next release. While waiting for the fix to be released, to workaround the issue please downgrade to either firebase-tools v13.13.1 or v13.13.0.

**Comment by joehan** [comment]
13.13.3 just released with a fix for this issue. Sorry for the inconvenience everyone!
